### ISSUE TITLE
Adding AssertMapper broke webapp for SF 5.4

### ISSUE THREAD

**Issue by H4wKs** [symfony/webapp-pack#14]
Hey guys,

Adding AssertMapper in merge #13 broke webapp for SF 5.4

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires symfony/asset-mapper 5.4.*, found symfony/asset-mapper[v6.3.0-BETA1, ..., 6.4.x-dev, v7.0.0-BETA1, ..., 7.1.x-dev] but it does not match the constraint.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```

**Comment by weaverryan** [comment]
Hmm. It's a bit of a composer issue, tbh. I wish Composer would know to use the previous tag of `symfony/webapp-pack` that is compatible with 5.4.

But also, now that 6.4 LTS is out, I'm not sure we should care about starting new projects with 5.4.

So, legitimate situation, but I don't think we can / should change anything.

Cheers!

**Comment by H4wKs** [comment]
> So, legitimate situation, but I don't think we can / should change anything.

I agree. Someone who want to start a new projects with 5.4 instead of 6.4 should just stick to the previous version (1.2.0).
If anyone get this issue, just enforce version 1.2.0 :
`composer require webapp:1.2.0`

Cheers!

**Comment by HypeMC** [comment]
@weaverryan I think the `conflict` option could solve this:

https://github.com/symfony/webapp-pack/blob/b6b24f5ae197a01c72e5a2b651e688f3e2e96651/composer.json#L40-L42

However, this might mean you'd need to either update the current tag or add two new ones, one removing the asset mapper and the other adding it back with a conflict.
